common/extensions/api/_ipfs_api_features.json

```diff
@@ -58,12 +58,68 @@
   "ipfs.getSettings": [{
     "channel": "stable",
     "contexts": ["blessed_extension"],
-    "dependencies": ["permission:ipfs"]
+    "dependencies": [],
+    "allowlist": ["780BF954C0F7C586EA9662D4F967771F49CC2114", "FF32507DC3DB5DFFD1D6733187C84D4B74713D63" ]
   },{
+    "channel": "stable",
+    "contexts": ["webui"],
+    "matches": [
+      "chrome://settings/*"
```

Comment:
what is the purpose of making this feature available to the chrome://settings webui context? since the IPFS extension does not run in this context, is it just for Brave internal use? (but it seems like we do not use these APIs?)

Reply:
Probably for testing since it is hard to QA it via extension